backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java

```diff
@@ -311,7 +311,7 @@ public Membership resolveGlobal(String qualifiedName) {
     @Override
     public Membership resolveLocal(String name) {
         // Try to resolve the Simple name in the current namespace
-        Optional<Membership> membership = this.getMembership().stream()
+        Optional<Membership> membership = this.visibleMemberships(new BasicEList<>(), false, false).stream()
```

Comment:
We had to use visibleMembership() at some point, but it could have been internal to getMembership() instead.

Reply:
That's not good, the specification does not allow to do that.
Furthermore, some existing tests fail with this modification.